src/gui/mzroll/tabledockwidget.h

```diff
@@ -20,320 +20,381 @@ class NumericTreeWidgetItem;
-     * If user press <save> button <addSameMzRtGroup> sets to true which will be used
-     * to add this group's corresponding compound name to already bookmarked group
-     * of same rt and mz value.
-    */
-    void acceptGroup();
-    /**
-     * @details-this is a slot tied with <save> button of prompt dialog <promptDialog>
-     * to show already bookmarked group with same mz and rt value.
-     * If user press <cancel> button <addSameMzRtGroup> sets to false which will be used
-     * to reject this group's corresponding compound name.
-    */
-    void rejectGroup();
-    void exportJson();
-    void showSelectedGroup();
-    void setGroupLabel(char label);
-    void showPeakGroup(int row);
-    void showLastGroup();
-    void showNextGroup();
-    void Train();
-    float showAccuracy(vector<PeakGroup *> &groups);
-    void saveModel();
-    void printPdfReport();
-    void updateTable();
-    void updateItem(QTreeWidgetItem *item);
-    void updateStatus();
-    //   void runScript();
-
-    void markGroupBad();
-    void markGroupGood();
-    bool checkLabeledGroups();
-    void markGroupIgnored();
-    void showAllGroups();
-    void showHeatMap();
-    /**
-     * @brief  modify name appropriate for xml attribute naming
-     * @detail This method makes sample name appropriate for using in attribute naming in mzroll file
-     * It is just replacing '#' with '_' and adding 's' for letting sample name start with english letter
-     * In future, if sample name has some other special character, we have to replace those also
-     * with appropriate character
-     * Error can be seen at compilation time
-    */
-    void cleanString(QString &name);
-    void showTreeMap();
-    void showScatterPlot();
-    void setClipboard();
-
-    void showConsensusSpectra();
-    void deleteGroups();
-
-    void deleteGroup(PeakGroup *groupX);
-    void sortBy(int);
-    void align();
-    void deleteAll();
-    void clusterGroups();
-    void findMatchingCompounds();
-    void showFiltersDialog();
-    void filterPeakTable();
-    int loadSpreadsheet(QString fileName);
-    int loadCSVFile(QString filename, QString sep);
-    void showMergeTableOptions();
-    void showMsgBox(bool check, int tableNo);
-    void mergeGroupsIntoPeakTable(QAction *action);
-    void switchTableView();
-
-    void setTableType(tableType t) { type = t; }
-    void setTableView(tableViewType t) { viewType = t; }
-    void clearClusters();
-
-    void writeQEInclusionList(QString fileName);
-    void writeMascotGeneric(QString fileName);
-    //void saveEICsJson(string filename);
-    //void saveEICJson(ofstream& out, EIC* eic);
-    vector<EIC *> getEICs(float rtmin, float rtmax, PeakGroup &grp);
-
-  protected:
-    void dragEnterEvent(QDragEnterEvent *event);
-    void dropEvent(QDropEvent *event);
-    void focusInEvent(QFocusEvent * event);
-    void focusOutEvent(QFocusEvent * event);
-
-  Q_SIGNALS:
-    void peakMarkedTableDock();
```

Comment:
@saifulbkhan This signal is not being used

Reply:
Right. will remove.